README.md

```diff
@@ -66,14 +66,14 @@ For example, with `wget`:
 
 ```
 cd ~
-wget https://github.com/digitalocean/doctl/releases/download/v1.16.0/doctl-<version>-linux-amd64.tar.gz
+wget https://github.com/digitalocean/doctl/releases/download/v1.17.0/doctl-<version>-linux-amd64.tar.gz
 ```
 
 Or with `curl`:
 
 ```
 cd ~
-curl -OL https://github.com/digitalocean/doctl/releases/download/v1.16.0/doctl-<version>-linux-amd64.tar.gz
+curl -OL https://github.com/digitalocean/doctl/releases/download/v1.17.0/doctl-<version>-linux-amd64.tar.gz
```

Comment:
```
curl -OL https://github.com/digitalocean/doctl/releases/download/v<version>/doctl-<version>-linux-amd64.tar.gz
```

where `<version>` is the full semantic version, e.g., `1.17.0`. 

Then we can stop updating the README each time.

Reply:
:+1: